app/models/work.rb

```diff
@@ -0,0 +1,25 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :category, inclusion: { in: ["album", "book", "movie"],
+                                    message: "category can only be album, book, or movie." }
+  validates :title, presence: true, uniqueness: true
+
+  has_many :votes
+  has_many :users, through: :votes
+
+  # def top_10_albums
+  #   albums = Work.where(category: "album")
+  #   return albums.order(votes: :desc).limit(10)
+  # end
```

Comment:
You should have made these class methods. 

```suggestion
   def self.top_10_albums
     albums = Work.where(category: "album")
     return albums.order(votes: :desc).limit(10)
   end
```